src/core/lib/slice/slice_internal.h

```diff
@@ -312,4 +312,21 @@ grpc_slice grpc_slice_from_moved_string(grpc_core::UniquePtr<char> p);
 // 0. All other slices will return the size of the allocated chars.
 size_t grpc_slice_memory_usage(grpc_slice s);
 
+void grpc_slice_buffer_transfer(grpc_slice_buffer* src, grpc_slice_buffer* dst);
```

Comment:
These new APIs are undocumented and their intended usage is extremely unclear.

Reply:
Updated with comments.